polynote-server/src/main/scala/polynote/server/SharedNotebook.scala

```diff
@@ -237,18 +239,59 @@ class IOSharedNotebook(
 
     def startInterpreterFor(id: CellID): IO[Stream[IO, Result]] = ensureKernel().flatMap(_.startInterpreterFor(id))
 
+
+    /**
+      * If the [[ResultValue]] has any [[UpdatingDataRepr]]s, create a [[SignallingRef]] to capture its updatese in a
```

Comment:
`s/updatese/updates`

Reply:
I think I actually meant "updateses", as in, "capture the updatses, precious!"

jk